### PR TITLE
Feature/add data

### LINKED ISSUE #44: Data Seeders
Update the data seed file to include new data set and has more items to work with 
Make sure that the products are correct

### PR DESCRIPTION
**Pull request by deadbelly**
### Fix/Feature/Bug?
Feature

### What does this PR do?
This adds data we can use for our survey and corresponding assets. It fixes a few small issues that came up while attempting to migrate as well.

### Where should review start?
`seeders/products` is the primary file that was changed. You can view the new data objects here. `public/images/stock` has also been fleshed out with a lot more images for us to use.

### How is this tested?
This hasn't been subjected to automated tests but we looked at the data in sql to ensure that it's all there. 

### Applicable tickets?
Closes #44

### CODE REVIEW

**Comment by deadbelly** [comment]
I'd love if one person on the front end and one person on the back end could each review this to makes sure we have what we need.

**Review by cunninghamge — COMMENTED** [review]
This looks good to me!

File: code/api/src/config/params.json [hunk 1]
```diff
@@ -9,8 +9,49 @@
...
+        "id": 1,
+        "name": "Shirt"
+      },
+      "bottoms": {
+        "id": 2,
+        "name": "Bottoms"
+      },
+      "jacket": {
+        "id": 3,
+        "name": "Jacket"
```
Comment (DrewBradley) [inline code/api/src/config/params.json:24]: Thanks. This paints a pretty clear picture for this!

**Review by DrewBradley — APPROVED** [review]
This looks good to me. The data is showing up as binary files in GH, but it looked right when we were discussing it earlier. Great work Boone!

**Pull request merged**